backEnd/src/main/java/dbMaintenance/handlers/AddOwnedGroupsCountHandler.java

```diff
@@ -0,0 +1,71 @@
+package dbMaintenance.handlers;
+
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.spec.UpdateItemSpec;
+import com.amazonaws.services.dynamodbv2.document.utils.ValueMap;
+import dbMaintenance.managers.MaintenanceDbAccessManager;
+import handlers.ApiRequestHandler;
+import java.util.Iterator;
+import javax.inject.Inject;
+import models.AppSettings;
+import models.Group;
+import models.User;
+import utilities.ErrorDescriptor;
+import utilities.Metrics;
+import utilities.ResultStatus;
+
+public class AddOwnedGroupsCountHandler implements ApiRequestHandler {
+
+  private MaintenanceDbAccessManager maintenanceDbAccessManager;
+  private Metrics metrics;
+
+  @Inject
+  public AddOwnedGroupsCountHandler(final MaintenanceDbAccessManager maintenanceDbAccessManager,
+      final Metrics metrics) {
+    this.maintenanceDbAccessManager = maintenanceDbAccessManager;
+    this.metrics = metrics;
+  }
+

```

Comment:
gimme dem docs

Reply:
Some day I'll do this with out being reminded. Today is not that day.